### PR TITLE
Use modal directly and change its options

### PR DESCRIPTION
**Pull request by leepeuker**

### CODE REVIEW

File: public/js/watchlist.js [hunk 1]
```diff
@@ -11,17 +12,23 @@ async function removeFromWatchList() {
             addAlert('watchlistAlert', 'Something has gone wrong, check the logs and please try again', 'danger');
             return;
         }
-        addAlert('watchlistAlert', 'Item has succesfully been removed from watchlist', 'success');
-        document.querySelector('i[data-watchlistEntryId="'+watchlistEntryId+'"]').parentElement.parentElement.remove();
+
+        location.reload()
```
Comment (leepeuker) [inline public/js/watchlist.js:16]: the current standard is to reload the page on success, I would like to keep this consistent. we can change this later when we decided on a new standard and adjust it everywhere

File: templates/component/modal-remove-from-watchlist.html.twig [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
             <div class="modal-header">
-                <h5 class="modal-title" id="removeFromWatchlistModalLabel">Remove item from watchlist</h5>
+                <h5 class="modal-title" id="removeFromWatchlistModalLabel"></h5>
                 <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
             </div>
             <div class="modal-body">
-                <button type="button" id="removeFromWatchlistBtn" onclick="removeFromWatchList()" class="btn btn-danger">Yes</button>
-                <button type="button" data-bs-dismiss="modal" class="btn btn-secondary">No</button>
+                <div class="d-grid gap-2">
+                    <button type="button" class="btn btn-primary" id="refreshImdbRatingButton" onclick="goToMovie()">Go to movie</button>
```
Comment (leepeuker) [inline templates/component/modal-remove-from-watchlist.html.twig:10]: not sure about `Got to movie`, but have no better idea right now

File: templates/component/modal-remove-from-watchlist.html.twig [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
             <div class="modal-header">
-                <h5 class="modal-title" id="removeFromWatchlistModalLabel">Remove item from watchlist</h5>
+                <h5 class="modal-title" id="removeFromWatchlistModalLabel"></h5>
                 <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
             </div>
             <div class="modal-body">
-                <button type="button" id="removeFromWatchlistBtn" onclick="removeFromWatchList()" class="btn btn-danger">Yes</button>
-                <button type="button" data-bs-dismiss="modal" class="btn btn-secondary">No</button>
+                <div class="d-grid gap-2">
+                    <button type="button" class="btn btn-primary" onclick="goToMovie()">Go to movie</button>
```
Comment (leepeuker) [inline templates/component/modal-remove-from-watchlist.html.twig:10]: not sure about `Go to movie`, but have no better idea right now

File: templates/component/modal-remove-from-watchlist.html.twig [hunk 4]
```diff
@@ -1,13 +1,15 @@
-<div class="modal fade modal-sm" id="removeFromWatchlistModal" tabindex="-1" aria-labelledby="removeFromWatchlistModal" aria-hidden="true">
-    <div class="modal-dialog">
+<div class="modal fade modal-sm" id="optionsModal" tabindex="-1" aria-labelledby="optionsModal" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered">
```
Comment (leepeuker) [inline templates/component/modal-remove-from-watchlist.html.twig:2]: all modals are centered, let us keep this consistent

File: templates/component/modal-remove-from-watchlist.html.twig [hunk 5]
```diff
@@ -1,13 +1,15 @@
...
         <div class="modal-content">
             <div class="modal-header">
-                <h5 class="modal-title" id="removeFromWatchlistModalLabel">Remove item from watchlist</h5>
+                <h5 class="modal-title"></h5>
```
Comment (leepeuker) [inline templates/component/modal-remove-from-watchlist.html.twig:5]: not sure if we need a title?

**Review by JVT038 — APPROVED** [review]
Looks good.

File: templates/component/modal-remove-from-watchlist.html.twig (hunk 3 above)
Comment (JVT038) [reply templates/component/modal-remove-from-watchlist.html.twig:10]: I would do something like 'View movie' or 'View movie details' .

**Pull request merged**